airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2667,8 +2667,94 @@ components:
             - failed
         reason:
           type: string
+    InvalidInputProperty:
```

Comment:
bummer that we have to lose the inheritance. i think it would be worthwhile leaving a comment explaining what the "required" fields of all exceptions are. maybe also worth mentioning why we are duplicating attributes as opposed to using inheritance.